ci-operator/step-registry/cucushift/hypershift-extended/idp-htpasswd/cucushift-hypershift-extended-idp-htpasswd-commands.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+export LC_CTYPE=C
+set -o nounset
+set -o errexit
+set -o pipefail
+
+export KUBECONFIG="${SHARED_DIR}/kubeconfig"
+count=$(oc get hostedclusters --no-headers --ignore-not-found -n clusters | wc -l)
+echo "$count"
+if [ "$count" -lt 1 ]  ; then
+    echo "namespace clusters don't have hostedcluster"
+    exit 1
+fi
+cluster_name=$(oc get hostedclusters -n clusters -o jsonpath='{.items[0].metadata.name}')
```

Comment:
Not sure we add oauth for all of hostedcluster or just the first one ?  If only the first one added, then we should always follow this rule for the guest cluster kubeconfig exposion.

Suggest add description like : "Limitation: we always & only select the first hostedcluster to add idp-htpasswd. "